java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Pass primary minion ip to the salt runner
```

Comment:
This entry can be a bit more descriptive with more context. Something like:
"Use primary minion IP when uploading images with 'kiwi-image-collect' runner"
(also for `susemanager-sls.changes`)

Reply:
Done